README.md

```diff
@@ -134,6 +144,20 @@ Next navigation:
   </cat-carousel>
 ```
 
+Center mode carousel:
+
+![](demo/center-mode.png)
+
+```HTML
+<cat-carousel
+    :items="items"
+    :item-per-page="5"
```

Comment:
I think @gilvent right about this one.
I think we can remove this one or ignore this props if centerMode.enable value is true